dojo/db_migrations/0219_system_settings_disclaimer_notif.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 5.0.8 on 2024-09-12 18:22
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dojo', '0218_system_settings_enforce_verified_status_and_more'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='system_settings',
+            old_name='disclaimer',
+            new_name='disclaimer_notifications',
```

Comment:
I don't know how common this is, but it might affects users setting the disclaimer via the api. A little note in the upgrade notes? It also affects people who are using customized notification templates.